### DISCUSSION TITLE
V8 API docs missing table of contents

### DISCUSSION THREAD

**Discussion by ajdiyassin** [TanStack/table#4022] [Ideas]
Hello, 

Currently, it's really hard to navigate the API on V8. I have to scroll up and down multiple times to figure out the API.

I think the docs are missing a "table of contents" feature which will give a general idea of the API without scrolling.

### Example from V7 docs:
![image](https://user-images.githubusercontent.com/103142364/173704673-2d721200-91b1-4a35-a2fa-4bb11f56707c.png)

**Comment by tannerlinsley** [comment]
Agreed. Wanna add it for me? The source for the docs is over on the @tanstack/tanstack.com repo.
